TACC/Frontera-Portal
# FP-475 create initial holding area

## Overview: ##

Create simple "hello world"-like components for holding area

## PR Status: ##



## Related Jira tickets: ##

* [FP-475](https://jira.tacc.utexas.edu/browse/FP-475)

## Summary of Changes: ##

## Testing Steps: ##
1.  with setup complete = false,  try to get to workbench and make sure you are redirected. 
2. with setup complete = true,  try to get to workbench and make sure you can. 

## Notes ##
@jchuahtacc ,  this just has 'hello-world'-like components and NOTE this also doesn't have the following thing from the JIRA
_Get the holding page to redirect user to workbench when it receives setup_complete=True event_
